client/src/pages/Login.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+//Material-ui imports
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import Link from '@material-ui/core/Link';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Background from '../elements/Background';
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    marginTop: theme.spacing(20),
+    padding: theme.spacing(5),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center'
+  },
+  headerText: {
+    marginTop: theme.spacing(10)
+  },
+  form: {
+    width: '100%',
+    marginTop: theme.spacing(3),
+    textAlign: 'center'
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+    padding: theme.spacing(1, 6),
+    border: 0,
+    borderRadius: 25,
+    color: '#fff',
+    textTransform: 'none',
+    fontSize: 15
+  }
+}));
+
+export default function SignIn() {
+  const classes = useStyles();
+
+  // DO WE LOG IN BY USERNAME OR EMAIL???
+  const [ loginUsername, setLoginUsername ] = useState('');
+  const [ loginPassword, setLoginPassword ] = useState('');
+  const [ submitClicked, setSubmitClicked ] = useState(false);
+
+  const loginHandler = (event) => {
+    event.preventDefault();
+    if (!loginUsername || !loginPassword) {
+      setSubmitClicked(true);
+    } else {
+      console.log(`Send login request to API ${loginUsername} ${loginPassword}`);
+    }
+    // Here goes login logic after we get server routes
+    // Axios({
+    //   method: 'POST',
+    //   data: {
+    //     username: loginUsername,
+    //     password: loginPassword
+    //   },
+    //   withCredentials: true,
+    //   url: 'http://localhost:3001/login'
+    // }).then((res) => {
+    //   if (res.data === 'Successfully Authenticated') {
+    //     window.location.href = '/'; //There must be a way to redirect with React, history.push() doesn't work correctly
+    //   } else {
+    //     console.log(res.data); //
+    //   }
+    // });
```

Comment:
This logic should be a dispatch to the useReducer hook

Reply:
For now its just early thoughts ( they are commented)

Reply:
Got it! Just want to make sure we're moving in the right direction with FE architecture.

Reply:
I really don't like pushing commented out code in pull requests - generally avoid that (keep notes on the side or use git history). Also when we make the API call, we will be using relative paths (not http://localhost:3001/login but /login)